### PR TITLE
remove unused value and fix spelling error

### PR DESCRIPTION
**Pull request by muxuelan**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/dynamiccontroller/application/application.go [hunk 1]
```diff
@@ -98,7 +98,6 @@ type Application struct {
 	count     uint64 // count the number of current citations
 	countLock sync.Mutex
 	tim       time.Time // record the last closing time of application, only make sense when count == 0
```
Comment (gy95) [inline cloud/pkg/dynamiccontroller/application/application.go:100]: you can fix this typo from `tim` to `time`

File: cloud/pkg/dynamiccontroller/application/application.go [hunk 2]
```diff
@@ -98,7 +98,6 @@ type Application struct {
...
-	//TODO: add lock
```
Comment (gy95) [inline cloud/pkg/dynamiccontroller/application/application.go:101]: better keep this comment? :) maybe definitely need to add a lock

Comment (muxuelan) [reply cloud/pkg/dynamiccontroller/application/application.go:101]: Aha, line-99 had already added lock.

File: cloud/pkg/dynamiccontroller/application/application.go (hunk 1 above)
Comment (muxuelan) [reply cloud/pkg/dynamiccontroller/application/application.go]: done

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**